libs/application-generic/src/services/feature-flags/process-env.service.ts

```diff
@@ -13,14 +12,7 @@ export class ProcessEnvFeatureFlagsService implements IFeatureFlagsService {
     this.isEnabled = false;
   }
 
-  async getBooleanFlag<T extends FeatureFlagsKeysEnum>({
-    key,
-    defaultValue,
-  }: IFeatureFlagContext<T>): Promise<boolean> {
-    return process.env[key] === 'true' || (defaultValue as boolean);
-  }
-
-  async getNumberFlag<T extends FeatureFlagsKeysEnum>({ key, defaultValue }: IFeatureFlagContext<T>): Promise<number> {
-    return Number(process.env[key]) || (defaultValue as number);
+  async getFlag<T_Result>(context: FeatureFlagContext<T_Result>): Promise<T_Result> {
+    return context.defaultValue;
```

Comment:
@djabarovgeorge we should return the process.env value before the default value.

Reply:
you are totally correct my miss, damn the auto-delete on save in the editor :(

Reply:
done